locallibrary/catalog/views.py

```diff
@@ -1,3 +1,24 @@
 from django.shortcuts import render
+from .models import Book, BookInstance, Author, Genre
 
-# Create your views here.
+
+def index(request):
+    num_books = Book.objects.all().count()
```

Comment:
Viết gọn lại thôi em nhen
```suggestion
    num_books = Book.objects.count()
```